include/deal.II/grid/reference_cell.h

```diff
@@ -354,6 +344,31 @@ namespace internal
 
 
 
+namespace ReferenceCells
+{
+  constexpr ReferenceCell Vertex =
+    internal::ReferenceCell::make_reference_cell_from_int(0);
+  constexpr ReferenceCell Line =
+    internal::ReferenceCell::make_reference_cell_from_int(1);
+  constexpr ReferenceCell Triangle =
+    internal::ReferenceCell::make_reference_cell_from_int(2);
+  constexpr ReferenceCell Quadrilateral =
+    internal::ReferenceCell::make_reference_cell_from_int(3);
+  constexpr ReferenceCell Tetrahedron =
+    internal::ReferenceCell::make_reference_cell_from_int(4);
+  constexpr ReferenceCell Pyramid =
+    internal::ReferenceCell::make_reference_cell_from_int(5);
+  constexpr ReferenceCell Wedge =
+    internal::ReferenceCell::make_reference_cell_from_int(6);
+  constexpr ReferenceCell Hexahedron =
+    internal::ReferenceCell::make_reference_cell_from_int(7);
+  constexpr ReferenceCell Invalid =
+    internal::ReferenceCell::make_reference_cell_from_int(
+      static_cast<std::uint8_t>(-1));
```

Comment:
```suggestion
  static DEAL_II_CONSTEXPR ReferenceCell Vertex =
    internal::ReferenceCell::make_reference_cell_from_int(0);
  static DEAL_II_CONSTEXPR ReferenceCell Line =
    internal::ReferenceCell::make_reference_cell_from_int(1);
  static DEAL_II_CONSTEXPR ReferenceCell Triangle =
    internal::ReferenceCell::make_reference_cell_from_int(2);
  static DEAL_II_CONSTEXPR ReferenceCell Quadrilateral =
    internal::ReferenceCell::make_reference_cell_from_int(3);
  static DEAL_II_CONSTEXPR ReferenceCell Tetrahedron =
    internal::ReferenceCell::make_reference_cell_from_int(4);
  static DEAL_II_CONSTEXPR ReferenceCell Pyramid =
    internal::ReferenceCell::make_reference_cell_from_int(5);
  static DEAL_II_CONSTEXPR ReferenceCell Wedge =
    internal::ReferenceCell::make_reference_cell_from_int(6);
  static DEAL_II_CONSTEXPR ReferenceCell Hexahedron =
    internal::ReferenceCell::make_reference_cell_from_int(7);
  static DEAL_II_CONSTEXPR ReferenceCell Invalid =
    internal::ReferenceCell::make_reference_cell_from_int(
      static_cast<std::uint8_t>(-1));
```

Reply:
`static` is awkward in header files, and I still want these objects to be `const`. Let's try if I can get away with just marking these objects as `DEAL_II_CONSTEXPR const` instead, or if that leads to linker errors for duplicate objects?